appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/core/BaseTransitionModel.kt

```diff
@@ -53,6 +53,14 @@ abstract class BaseTransitionModel<NavTarget, ModelState>(
         removeDestroyedElements()
     }
 
+    override fun onAnimationFinished(navElement: NavElement<NavTarget>) {
```

Comment:
TODO TransitionModel doesn't know about navElement

Reply:
```suggestion
    override fun cleanupState(navElement: NavElement<NavTarget>) {
```

I feel this one shouldn't mention animations, just focus on its own functionality.